client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -163,7 +163,7 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
           onChange(newNum);
         }}
         onBlur={() => {
-          onChange(Number(parse(textValue ?? '')) || undefined);
+          onChange(Number(parse(textValue ?? '')) ?? undefined);
```

Comment:
Not strictly part of this PR, but fixes a mistake which wipes out a `0` entry in number input. (That darned `0 == false` thing again 🙄)